src/components/program-progress/ProgramProgressCourses.jsx

```diff
@@ -108,22 +148,55 @@ const ProgramProgressCourses = ({ courseData }) => {
       {coursesInProgress?.length > 0
       && (
         <div className="mb-5">
-          <h4 className="white-space-pre">COURSES IN PROGRESS    {coursesInProgress.length}</h4>
+          <h4 className="white-space-pre">
+            <FormattedMessage
+              id="enterprise.dashboard.programs.about.page.courses.in.progress"
+              defaultMessage="COURSES IN PROGRESS    {count}"
```

Comment:
@muhammad-ammar Yes, the space was intentional as it was present in the original string and I retained it in the marked string. 
However, I have moved the space outside the `FormattedMessage` since it is required.
<img width="730" alt="Screenshot 2024-02-23 at 12 59 16 PM" src="https://github.com/openedx/frontend-app-learner-portal-enterprise/assets/113524403/52f42b3b-5319-40d5-ae3d-22c1cca3e241">